sample/src/main/res/layout/fragment_dialogs.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.appcompat.widget.LinearLayoutCompat
```

Comment:
Some small feedbacks : 
- Can you use the padding screen to be aligned with the other screens? 
- Maybe use title or description and only a "Show dialog" button in each section to have a screen more readable? Can be discussed together 😉